BUILDING.md

```diff
@@ -46,6 +46,30 @@ make setup -j$(nproc) OPTFLAGS=-O2 DEBUG=0
 make -j $(nproc) OPTFLAGS=-O2 DEBUG=0
 ```
 
+## macOS
+
+1. Requires `gcc@12, sdl2, libpng, glew, spdlog, pulseaudio, x11` (can be installed via brew, etc)
+```bash
+# Clone the repo
+git clone git@github.com:HarbourMasters/ShipWright.git
+cd ShipWright
+# Copy the baserom to the OTRExporter folder
+cp <path to your ROM> OTRExporter
+# Clone and build StormLib
+git clone https://github.com/ladislav-zezula/StormLib external/StormLib
+cmake -B external/StormLib/build -S external/StormLib
+cmake --build external/StormLib/build
+cp external/StormLib/build/libstorm.a external
+# Copy glew lib to external (path might be different depending on how you installed it)
+cp /usr/local/lib/libGLEW.a external
```

Comment:
```suggestion
cp `brew --prefix`/lib/libGLEW.a external
```

On M1 libGLEW is available at /opt/homebrew/lib/libGLEW.a

Reply:
this step is likely not needed, it's done via pkg-config to find the right .a file

Reply:
Yes, you are correct. I skipped this step and it still compiled

Reply:
Will remove this step!